package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "moment": "^2.22.2",
```

Comment:
Nice!

Reply:
@Mullefa thanks for your comments on my PR, and for taking the time to review my work